cartography/data/jobs/analysis/gcp_compute_firewall_analysis.json

```diff
@@ -0,0 +1,15 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (fw:GCPFirewall{unrestricted_access:true})<-[:RESOURCE]-(vpc:GCPVpc)<-[:RESOURCE]-(:GCPProject{id: $GCP_PROJECT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WITH fw LIMIT $LIMIT_SIZE REMOVE fw.unrestricted_access return COUNT(*) as TotalCompleted",
```

Comment:
Change here for `exposed_internet` and `exposed_internet_type`

Reply:
this is not done yet

Reply:
done